src/towncrier/test/test_create.py

```diff
@@ -249,3 +249,35 @@ def test_create_orphan_fragment_custom_prefix(self, runner: CliRunner):
         self.assertEqual(len(change.stem), 11)
         # Check the remainder are all hex characters.
         self.assertTrue(all(c in string.hexdigits for c in change.stem[3:]))
+
+    @with_isolated_runner
+    def test_in_different_dir_with_nondefault_newsfragments_directory(self, runner):
+        """
+        Config location differs from the base directory for news file and fragments.
+
+        This is useful when multiple projects share one towncrier configuration.
+        """
+        Path("pyproject.toml").write_text(
+            # Important to customize `config.directory` because the default
```

Comment:
I don't understand this comment here.

Maybe, it's important to note that this is a configured using a sub-directory that is later combined with the CLI argument.

Also, maybe it's also important to pass an empty `project` value for this test